### DISCUSSION TITLE
Moving items from one named bag to another

### DISCUSSION THREAD

**Discussion by DrewBrasher** [OrchardCMS/OrchardCore#10046] [Q&A] [answered]
Is it possible to move an item from one named bag to another from the admin menu?  If not from the admin menu what about from code?

For example, if you have a content type that has 2 named bag parts called "Section One" and "Section Two".  Both take content items of the same type.  I add an item to Section One, then later decide it should be in Section Two.

**Comment by Skrypt** [comment, marked as answer]
You will need to execute some SQL queries on your Document table to morph the JSON in the Content column. Search for SQL JSON Functions. The Named BagParts stores it's own data in the main content item JSON data. It basically allow to contain some more JSON data on a single content item (document). Moving data from one Named BagPart to the other should consist basically about changing the JSON Named BagPart name from "Section One" to "Section Two" of the JSON node.

You can't actually do this from the Admin UI. From code it could be done with some migration.

**Reply by Skrypt** [reply]
Here is an example of JSON data with a "Portfolio" Named BagPart : 

![image](https://user-images.githubusercontent.com/3228637/128532189-0d9c5fba-f3af-4198-bf33-20f985097feb.png)


You basically need to change this "Portfolio" to "SomethingElse" ...

**Reply by DrewBrasher** [reply]
Thank you.  I may just use a list or single bag and then separate items out into sections in the display template based on a content picker or category field instead but this is good to know.
